core/src/test/java/org/elasticsearch/test/ESIntegTestCase.java

```diff
@@ -1882,7 +1882,7 @@ public static DateTimeZone randomDateTimeZone() {
     /**
      * Returns path to a random directory that can be used to create a temporary file system repo
      */
-    public Path randomRepoPath() {
+    public static Path randomRepoPath() {
```

Comment:
Yeah - at first I tried to go down the path of making everything static. I can just revert this though.